godtools/App/Features/Tools/SwiftUI/AllTools/AllToolsContentViewModel.swift

```diff
@@ -21,10 +21,13 @@ class AllToolsContentViewModel: NSObject, ObservableObject {
     private let favoritedResourcesCache: FavoritedResourcesCache
     private let favoritingToolMessageCache: FavoritingToolMessageCache
     private let analytics: AnalyticsContainer
+        
+    private var spotlightViewModel: ToolSpotlightViewModel?
```

Comment:
Good call! For some reason I was thinking it would error because `dataDownloader` and all the other params needed to be initialized first.  But that makes sense that if it's `lazy`, the compiler knows all the instance variables will exist already.